gamemodes/terrortown/gamemode/shared/sh_sql.lua

```diff
@@ -233,11 +233,26 @@ end
 -- @param table keys the keys for the data @{table}
 -- @return boolean Whether the database table was created successfully
 -- @realm shared
--- @todo usage
+-- @deprecated
 function sql.CreateSqlTable(tableName, keys)
-    local result
-
-    if not sql.TableExists(tableName) then
+    Dev(
+        2,
+        "`sql.CreateSqlTable` is deprecated and should be handled by migrations.",
+        "This ran for table:",
+        tableName
+    )
```

Comment:
Deprecating this functionality idk.
Could be still used in migration files.
Maybe check a flag, so that it works without dev message in migrations?

Reply:
Writing own sql statements directly in migration files seems to be the best way.